src/coreclr/vm/interpexec.cpp

```diff
@@ -1105,6 +1152,27 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                 memset(LOCAL_VAR(ip[1], void*), 0, ip[2]);
                 ip += 3;
                 break;
+            case INTOP_LOCALLOC:
+            {
+                int32_t len = LOCAL_VAR(ip[2], int32_t);
+                void* mem;
+
+                if (len > 0)
+                {
+                    len = ALIGN_UP(len, INTERP_STACK_ALIGNMENT);
+                    mem = pThreadContext->pFrameDataAllocator->Alloc((InterpreterFrame*)pFrame, len);
+                    if (mem != NULL && pMethod->initLocals)
```

Comment:
This should never return back. It should either allocate memory, or fail with stackoverflow exception. Handling of NULL should be exception TODO.